package.json

```diff
@@ -22,13 +24,32 @@
     "grunt-mocha": "^1.0.2",
     "grunt-postcss": "~0.8.0",
     "grunt-sass": "~2.0.0",
+    "karma": "^1.7.0",
+    "karma-mocha": "^1.3.0",
+    "karma-phantomjs-launcher": "^1.0.4",
+    "karma-sourcemap-loader": "^0.3.7",
+    "karma-webpack": "^2.0.4",
     "load-grunt-tasks": "~3.5.2",
     "mocha": "^3.0.2",
+    "node-sass": "^4.5.3",
     "pixrem": "~3.0.2",
+    "sass-loader": "^6.0.6",
+    "style-loader": "^0.18.2",
     "time-grunt": "~1.4.0",
+    "vue-loader": "^12.2.2",
+    "vue-style-loader": "^3.0.1",
+    "vue-template-compiler": "^2.4.1",
     "webpack": "^3.2.0"
   },
   "engines": {
     "node": ">=0.8.0"
+  },
+  "dependencies": {
+    "bootstrap": "^3.3.7",
```

Comment:
Hmmm this was probably added automatically? We use bootstrap 4 - which is already installed by bower. If it needs to be installed here too (probably), it should be the 4 version?